behind-the-veil-siteroot/imports/ui/components/card/ServiceCard.jsx

```diff
@@ -72,8 +72,8 @@ export const ServiceCard = ({
                 {/* button to specific booking detail page*/}
                 <Button className="flex flex-row gap-x-2 justify-center items-center w-4/5 lg:w-1/2 min-w-40
                 bg-secondary-purple hover:bg-secondary-purple-hover transition duration-500"
-                        onClick={() => navigateTo('/services/' + serviceId)}>
-                    <CalendarDaysIcon className="h-6 w-6 min-h-6 min-w-6"/>
+                        onClick={() => navigateTo('/service/' + serviceId)}>
+                    <CalendarDaysIcon className="size-6 stroke-2"/>
```

Comment:
'size-6 stroke-2' appears quite often. If you reckon it's worth it, we could define a class in the components layer in the main.css file, smth like:
```
icon-base {
    @apply size-6 stroke-2;
}
```
and then use this icon-base class for icons instead of 'size-6 stroke-2'

Up to you if you want, otherwise just resolve this comment

Reply:
sounds good will do